app/views/users/show.html.erb

```diff
@@ -1 +1,25 @@
-<h1>Specefic User Info</h1>
\ No newline at end of file
+<section id="body-content" class="container">
+    <%= render partial: 'shared/user', locals: { user: @user } %>
+    <div class="row mt-4">
+        <div class="col-lg-12 border border-dark">
+            <h4>Bio</h4>
+            <p class="mt-2"><%= @user.bio %></p>
+        </div>
+    </div>
+
+    <% @posts.each_with_index do |post, index| %>  
+        <div class="row mt-4">
+            <div class="col-lg-12 border border-dark">
+                <h4 class="pt-2">Post #<%= index + 1 %></h4>
+                <p class="mt-2"><%= truncate(post.text, length: 200, omission: '...') %></p>
+                <p class="d-flex justify-content-end">Comments <%= post.comments_counter %>, Likes <%= post.likes_counter%></p>
+            </div>
+        </div>
+    <% end %>
```

Comment:
- [ ] Please use the method `recent_posts`  available in the `user` model and loop through it instead of looping through all posts. Also, the `posts #` should be a link to the post-show page.